static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -708,12 +708,20 @@ svg.main-menu__icon--social__svg {
 
 .tertiary-navigation__item {
     width: calc(50% - #{$gs-gutter / 2});
+    height: $gs-row-height;
     border-top: 1px solid $neutral-5;
+    padding-top: 2px;
     padding-bottom: $gs-baseline / 2;
 }
 
+.tertiary-navigation__text {
+    width: calc(100% - #{$gs-gutter});
+    float: left;
+}
+
 .tertiary-navigation__link {
     @include fs-textSans(5);
+    line-height: 18px;
```

Comment:
That is an interesting though @gtrufitt . I don't think the line height is related to the item height though. If that variable value was ever changed so that `$gs-row-height` was equal to 40px instead, we would still want the line-height to be 18px.

That said, I doubt we would change the value of  `$gs-row-height` anytime soon, if ever.